.github/workflows/build.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Get version
+        id: version
+        uses: home-assistant/actions/helpers/version@master
+
+      - name: Patch manifest and zip
+        run: |
+          sed -i 's/v0.0.0/${{ steps.version.outputs.version }}/g' custom_components/scheduler/manifest.json
```

Comment:
`/g` can be removed. With that flag, you are doing a global replace (all occurrences). I think we should only replace the first occurrence (just in case)

Reply:
Should I remove `/g` or just the `g`?

Reply:
@KTibow you should remove just the `g`. According to https://linuxize.com/post/how-to-use-sed-to-find-and-replace-string-in-files/

Reply:
```suggestion
          sed -i 's/v0.0.0/${{ steps.version.outputs.version }}/' custom_components/scheduler/manifest.json
```